python_dwd/metadata_dwd.py

```diff
@@ -133,7 +133,7 @@ def metadata_for_dwd_data(parameter: Parameter,
         metainfo = metaindex_for_1minute_data(parameter=parameter,
                                               time_resolution=time_resolution)
 
-    if all(np.isnan(metainfo[STATE_NAME])):
+    if all(pd.isnull(metainfo[STATE_NAME])):
```

Comment:
python guidelines says: do not ask if you can do something just do it and ask for excuse later. So I would suggest an exception here